panel/chat/feed.py

```diff
@@ -231,8 +231,8 @@ def __init__(self, *objects, **params):
 
         # forward message params to ChatMessage for convenience
         message_params = params.get("message_params", {})
-        for param_key in list(params.keys()):
-            if param_key not in ChatFeed.param and param_key in ChatMessage.param:
+        for param_key in params.copy():
```

Comment:
I would usually just do:

```suggestion
        for param_key in list(params):
```

No strong preference.

Reply:
I follow this tip: https://twitter.com/raymondh/status/1251953119556104192

![image](https://github.com/holoviz/panel/assets/19758978/c2970634-7bba-4c0b-b137-ca3cfccbfa47)

Reply:
`copy` is faster, so you've converted me:

<img width="610" alt="Screenshot 2024-01-31 at 18 31 42" src="https://github.com/holoviz/panel/assets/1550771/c2bb2592-1766-4523-8dbd-f1cd29edf025">

Reply:
Should have known, lists are slow, dicts are fast.

Reply:
What if the dict size is much larger? 100k?

Reply:
There does appear to be a cross-over point, somewhere near 8M on my machine:

<img width="573" alt="Screenshot 2024-01-31 at 18 35 19" src="https://github.com/holoviz/panel/assets/1550771/df24071c-f9c8-454c-bbe0-c034acbc7d64">